### PR TITLE
Added test cases to increase coverage for votemanager

### LINKED ISSUE #270: Increase Test Coverage on Vote Manager

### PR DESCRIPTION
**Pull request by GauravJain9**
fixes #270 
1)  Added some test cases to increase the coverage of statements and functions 
2)  Changed one line which was already in test case  (https://github.com/razor-network/contracts/pull/271/files#diff-f683c8d5b95777f9e626d97eb2b74a82ffe9f58b93fe5f4a6a704f90885120baL163) 
     Reason : Sometimes it was passing and some times failing due to which it makes another 2 3 test cases fail , now it is fixed

Everything is working fine locally but tests are not passing during checks here (reason : unknown ).

### CODE REVIEW

File: test/VoteManager.js [hunk 1]
```diff
@@ -65,13 +65,13 @@ describe('VoteManager', function () {
         );
         const tx = voteManager.connect(signers[5]).commit(epoch, commitment1);
 
-        await assertRevert(tx, 'Contract should be initialized');
+        assertRevert(tx, 'Contract should be initialized');
```
Comment (SkandaBhat) [inline test/VoteManager.js:68]: Why was await removed?

File: test/VoteManager.js [hunk 2]
```diff
@@ -65,13 +65,13 @@ describe('VoteManager', function () {
...
       });
 
       it('should not be able to initiliaze VoteManager contract without admin role', async () => {
         const tx = voteManager.connect(signers[1]).initialize(stakeManager.address, rewardManager.address, blockManager.address,
           parameters.address, assetManager.address);
-        await assertRevert(tx, 'AccessControl');
+        assertRevert(tx, 'AccessControl');
```
Comment (SkandaBhat) [inline test/VoteManager.js:74]: Same as above

File: test/VoteManager.js [hunk 3]
```diff
@@ -129,6 +131,20 @@ describe('VoteManager', function () {
...
+        const votes = [100, 200, 300, 400, 500, 600, 700, 800, 900];
+        const tree = merkle('keccak256').sync(votes);
+        const root = tree.root();
+        const commitment1 = utils.solidityKeccak256(
+          ['uint256', 'uint256', 'bytes32'],
+          [epoch, root, '0x727d5c9e6d18ed15ce7ac8d3cce6ec8a0e9c02481415c0823ea49d847ccb9ddd']
+        );
+
+        const tx = voteManager.connect(signers[3]).commit(epoch, commitment1);
+        assertRevert(tx, 'already commited');
```
Comment (SkandaBhat) [inline test/VoteManager.js:145]: Same as above

File: test/VoteManager.js [hunk 4]
```diff
@@ -157,16 +173,16 @@ describe('VoteManager', function () {
         let tx = voteManager.connect(signers[3]).reveal(epoch, tree.root(), assignedAssetsVotesCopy, proof,
           '0x727d5c9e6d18ed15ce7ac8d3cce6ec8a0e9c02481415c0823ea49d847ccb9ddd',
           signers[3].address);
-        await assertRevert(tx, 'Revealed assets not equal to required assets per staker');
+        assertRevert(tx, 'Revealed assets not equal to required assets per staker');
```
Comment (SkandaBhat) [inline test/VoteManager.js:176]: Same as above

File: test/VoteManager.js [hunk 5]
```diff
@@ -157,16 +173,16 @@ describe('VoteManager', function () {
...
         // Incorrect Reveal
-        assignedAssetsVotesCopy.splice(0, 1);
+        assignedAssetsVotesCopy.splice(5, 6);
         const assetId = await findAssetNotAlloted(assignedAssetsVotesCopy, votes.length);
         assignedAssetsVotesCopy[0].id = assetId;
         tx = voteManager.connect(signers[3]).reveal(epoch, tree.root(), assignedAssetsVotesCopy, proof,
           '0x727d5c9e6d18ed15ce7ac8d3cce6ec8a0e9c02481415c0823ea49d847ccb9ddd',
           signers[3].address);
-        await assertRevert(tx, 'Revealed asset not alloted');
+        assertRevert(tx, 'Revealed asset not alloted');
```
Comment (SkandaBhat) [inline test/VoteManager.js:185]: Same as above

**Review by SkandaBhat — CHANGES_REQUESTED** [review]

**Comment by SkandaBhat** [comment]
CI tests are failing + lot of await keywords missing. Please fix them.

File: test/VoteManager.js (hunk 1 above)
Comment (GauravJain9) [reply test/VoteManager.js:68]: await is no longer required dicussed yesterday @hrishikeshio told to remove await keyword. And it is not making any difference. Also all test cases are passing locally on my terminal. I can't really figure out why are they failing. 
I will Check again.
@SkandaBhat

Comment (GauravJain9) [reply test/VoteManager.js:68]: Promises are being handled

**Comment by GauravJain9** [comment]
> CI tests are failing + lot of await keywords missing. Please fix them.

@SkandaBhat  Have fixed all the issues

**Review by SamAg19 — APPROVED** [review]

**Review by abhishekvispute — APPROVED** [review]

**Pull request merged**